plates/plate_cell.py

```diff
@@ -0,0 +1,55 @@
+from dimensions import Dimensions
+from quadrants import Quadrants
+
+LETTERS_CONSTANT = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 'AA', 'AB', 'AC', 'AD', 'AE', 'AF']
+
+class PlateCell:
+
+    def __init__(self, cell_number, dimensions, quadrants):
+        self.dimensions = dimensions
+        self.quadrants = quadrants
```

Comment:
I don't think PlateCell needs a quadrant.. You need it only in the conversion to higher dimensions.